Documentation/gettingstarted/istio.rst

```diff
@@ -4,6 +4,8 @@
     Please use the official rendered version released here:
     https://docs.cilium.io
 
+.. _gsg_istio:
```

Comment:
Unintended change?

Reply:
No, it's used on Documentation/contributing/development/dev_setup.rst on line 480